arcgis-ios-sdk-samples/Augmented reality/Navigate in AR/README.md

```diff
@@ -52,13 +52,17 @@ When you start, route instructions will be displayed and spoken. As you proceed
 
 This sample uses Esri's [World Elevation](https://elevation3d.arcgis.com/arcgis/rest/services/WorldElevation3D/Terrain3D/ImageServer) service to ensure that route lines are placed appropriately in 3D space. It uses Esri's [World Route](https://www.arcgis.com/home/item.html?id=1feb41652c5c4bd2ba5c60df2b4ea2c4) service to calculate routes. The world routing service requires authentication and does consume ArcGIS Online credits.
 
-**Note:** This item requires an ArcGIS Online organizational subscription or an ArcGIS Developer account and consumes credits.
+**Note:** This routing service requires one of the following authentication methods
 
-To access this sample, you'll need to do one of the following:
+* An ArcGIS Online organizational subscription and consumes credits
+* An ArcGIS Developer account and consumes credits
+* An API key
 
-* Sign in with an account that is a member of an organizational subscription.
-* Sign in with a developer account.
-* Register an application and use your application's credentials.
+To access this sample with ArcGIS identity/named user login using OAuth workflow, please read more at [ArcGIS Identity](https://developers.arcgis.com/documentation/mapping-apis-and-services/security/arcgis-identity/).
+
+The routing service can also be accessed with an API key, either specified in AppDelegate or in the sample. Please read more at [API keys](https://developers.arcgis.com/documentation/mapping-apis-and-services/security/api-keys/).
```

Comment:
This reference two things that users may not have: 1) `AppDelegate` (their app delegate class may not be named that or they may not have an app delegate if using SwiftUI), and 2) the sample (their app is not going to be a sample viewer). Could that part just be taken out?

```suggestion
The routing service can also be accessed with an API key. Please read more at [API keys](https://developers.arcgis.com/documentation/mapping-apis-and-services/security/api-keys/).
```